### PR TITLE
refactor api handler

### PR DESCRIPTION
**Pull request by s7u4rt99**

### CODE REVIEW

File: commands/task_command.py [hunk 1]
```diff
@@ -214,22 +208,16 @@ async def task_schedule_updated(update: Update, context: ContextTypes.DEFAULT_TY
...
 async def task_command_end(update: Update, context: ContextTypes.DEFAULT_TYPE):
-    data = {
-        "userId": 123,
-        "name": "",
-        "description": "",
-    }
-
-    url_post = f"{os.getenv('REQUEST_URL')}/tasks"
-    requests.post(url_post, json=data)
+    add_task(userId=123, title="", description="")
```
Comment (s7u4rt99) [inline commands/task_command.py]: @joshuan98 help me fill up with the `userId`, `title` and `description`.

File: commands/task_command.py [hunk 2]
```diff
@@ -151,14 +152,7 @@ async def task_schedule_no_update(update: Update, context: ContextTypes.DEFAULT_
...
 
-    data = {
-        "userId": 123,
-        "name": title,
-        "description": "",
-    }
-
-    url_post = f"{os.getenv('REQUEST_URL')}/tasks"
-    requests.post(url_post, json=data)
+    add_task(userId=123, title=title, description="")
```
Comment (s7u4rt99) [inline commands/task_command.py]: @joshuan98 help me fill up with the `userId`, `title` and `description`.

File: commands/event_command.py [hunk 3]
```diff
@@ -125,14 +124,7 @@ async def event_command_end(update: Update, context: ContextTypes.DEFAULT_TYPE):
...
 
-    data = {
-        "userId": 123,
-        "name": title,
-        "description": "",
-    }
-
-    url_post = f"{os.getenv('REQUEST_URL')}/tasks"
-    requests.post(url_post, json=data)
+    add_task(userId=123, title=title, description="")
```
Comment (s7u4rt99) [inline commands/event_command.py]: @joshuan98 help me fill up with the `userId`, `title` and `description`.

Comment (dZhei98) [reply commands/event_command.py]: title is above, no description, userId = context.chat_data["user_id"]

**Pull request merged**